src/storage/disk_table.h

```diff
@@ -39,6 +39,10 @@
 #include "rocksdb/utilities/checkpoint.h"
 #include "storage/iterator.h"
 #include "storage/table.h"
+#include "base/glog_wapper.h"  // NOLINT
```

Comment:
- reorder the header. move the below of `#include "base/endianconv.h"`
- remove `NOLINT`